TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/Finger.java

```diff
@@ -0,0 +1,61 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+import com.qualcomm.robotcore.hardware.Servo;
+
+class Finger {
+
+    private final double openPosition;
+    private final double closePosition;
+    private final Servo servo;
+    private boolean isOpen;
+
+    protected Finger(Servo fingerServo, double fingerOpenPosition, double fingerClosePosition) {
+        this.servo = fingerServo;
+        this.openPosition = fingerOpenPosition;
+        this.closePosition = fingerClosePosition;
+    }
+
+    protected Finger(Finger other) {
+        this.servo = other.getServo();
+        this.openPosition = other.getOpenPosition();
+        this.closePosition = other.getClosePosition();
+    }
+
+    private void setPosition(double position) {
+        servo.setPosition(position);
+    }
+
+    protected Servo getServo() {
```

Comment:
no. you shouldn't be able to access the servo from outide the finger

Reply:
fair